Library/Homebrew/utils/bottles.rb

```diff
@@ -0,0 +1,104 @@
+require "tab"
+require "extend/ARGV"
+
+module Utils
+  class Bottles
+    class << self
+      def tag
+        @bottle_tag ||= "#{ENV["HOMEBREW_SYSTEM"]}-#{ENV["HOMEBREW_PROCESSOR"]}".downcase.to_sym
+      end
+
+      def built_as?(f)
+        return false unless f.installed?
+        tab = Tab.for_keg(f.installed_prefix)
+        tab.built_as_bottle
+      end
+
+      def file_outdated?(f, file)
+        filename = file.basename.to_s
+        return unless f.bottle && filename.match(Pathname::BOTTLE_EXTNAME_RX)
+
+        bottle_ext = filename[bottle_native_regex, 1]
```

Comment:
`bottle_native_regex` has been renamed to `native_regex` (or `Utils::Bottles::native_regex`). See a few lines below.